setup.cfg

```
@@ -69,7 +69,8 @@ all =
     tensorflow > 1.10, < 2.3
     ; Version capped due to tensorflow incompatibility
     protobuf < 4
-    matplotlib
+    ; Version capped due to shap incompatibility
+    matplotlib < 3.6.0
```

Consider using 

```suggestion
    matplotlib != 3.6.0
```

if this seems like a matplotlib issue that will be fixed soon.